pkgs/by-name/ne/nexus-mods-app/package.nix

```diff
@@ -0,0 +1,125 @@
+{ _7zz
+, buildDotnetModule
+, copyDesktopItems
+, desktop-file-utils
+, dotnetCorePackages
+, fetchFromGitHub
+, fontconfig
+, lib
+, libICE
+, libSM
+, libX11
+, nexus-mods-app
+, runCommand
+, enableUnfree ? false # Set to true to support RAR format mods
+}:
+let
+  pname = "nexus-mods-app";
+  version = "0.4.1";
```

Comment:
Bump to 0.5?

Reply:
No, I'm trying to get the update script to work to prove it with a 0.5 upgrade.